### DISCUSSION TITLE
eas update --branch production not working

### DISCUSSION THREAD

**Discussion by erictangborn20** [community/community#124683] [Programming Help]
### Body

I'm using expo with react native. I've been updating my app with eas update --branch production and it's always updated the app in the past. I ran it yesterday and it said and it was successful, but the updates are not there when I open the app. The only difference is that I updated from expo49 to expo51.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by luminaire-dev** [comment]
I wonder if this is a caching issues. Can you try clearing the cache on your device or simulator and then reopening the app to see if the updates are applied.

**Comment by Wilsmac** [comment]
Gracias.

**Comment by Sworzen1** [comment]
Eas update can not update ur native layer like: app config, app icon, splash screen etc..
More about you can find here: [How EAS update works](https://docs.expo.dev/eas-update/how-it-works/)
If you still have the problem, please provide more context :)
